packages/rolldown/tests/fixtures/output/format/iife/exports/_config.ts

```diff
@@ -13,6 +13,8 @@ export default defineTest({
     expect(output.output[0].code).toMatchInlineSnapshot(`
       "(function(exports, node_path) {
 
```

Comment:
Could you also fix this empty line in another PR?

Reply:
Fixed.